### ISSUE TITLE
Extract maintainers from docstrings to separate page

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7224]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)| @puzzlement reported|
|-|-|
|Trac ID|trac#494|
|Type|enhancement|
|Created|2004-02-09 04:40:27Z|

<details><summary>Searchable metadata</summary>

```
trac-id__494 494
type__enhancement enhancement
reporter__hypatia hypatia
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__website website
keywords__ 
time__1076301627000000 1076301627000000
changetime__1471515079316695 1471515079316695
version__ 
owner__ 
cc__spiv cc__hypatia cc__tekNico
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="puzzlement's avatar" src="https://avatars.githubusercontent.com/u/94942?s=50" width="50" height="50">](https://github.com/puzzlement)|@puzzlement commented|
|-|-|

```
#!html
<pre>
It would be good to have a single page listing the maintainers of packages
and/or modules, ideally generated from the docstrings of various packages.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="teknico's avatar" src="https://avatars.githubusercontent.com/u/172521?s=50" width="50" height="50">](https://github.com/teknico)|@teknico set owner to @teknico|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="teknico's avatar" src="https://avatars.githubusercontent.com/u/172521?s=50" width="50" height="50">](https://github.com/teknico)|@teknico set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix commented|
|-|-|

This appears to be done at TwistedProjects. Can it be closed?

**Comment by twisted-trac** [comment]
|[<img alt="teknico's avatar" src="https://avatars.githubusercontent.com/u/172521?s=50" width="50" height="50">](https://github.com/teknico)|@teknico set status to `closed`|
|-|-|

Right, sorry, I did it, but forgot the ticket.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_7"></a>|@rodrigc commented|
|-|-|

In changeset f6766ae23afdf36e56a3c978d0f47ff34670f250
```
#!CommitTicketReference repository="" revision="f6766ae23afdf36e56a3c978d0f47ff34670f250"
Merge pull request #494 from twisted/8755-rodrigc-stdlib-py3

Author: rodrigc
Reviewer: acabhishek942
Fixes: #8755

Fix test_stdlib.py on Windows with Python 3.5
```
